master/SendImages.py

```diff
@@ -0,0 +1,39 @@
+import requests
+import argparse
+import random
+
+from DataBase.Redis import Redis
+
+class SendImages:
```

Comment:
It's a small nitpicking: It will be better to not have verbs as your class names, as we don't expect classes to _do_ something; Doing the actual job is done by functions. For example, you could rename this to `ImageSender` or something like that.

The main reference is: https://peps.python.org/pep-0008/